tests/functional/prometheous/test_prometheus_validations_file.py

```diff
@@ -333,3 +335,66 @@ def test_prometheus_pv_resize(self):
             pod_name=pod_name, pvc_name=pvc_name,
             filename="secondfilename", dirname="seconddirname",
             metric_data=resize_metrics, operation="create")
+
+    @pytest.mark.tier3
+    def test_prometheus_volume_metrics_on_node_reboot(self):
+        """Validate volume metrics using prometheus before and after node
+        reboot"""
+
+        # Create I/O pod with PVC
+        pvc_name = self.create_and_wait_for_pvc()
+        pod_name = openshift_ops.oc_create_tiny_pod_with_volume(
+            self._master, pvc_name,
+            "autotest-{}".format(utils.get_random_str()),
```

Comment:
create random string at the start as below and use same variable for all the places in test -
```
prefix = "autotest-{}".format(utils.get_random_str())
```

Reply:
done